pootle/apps/pootle_store/constants.py

```diff
@@ -73,3 +73,7 @@
     UNTRANSLATED: "untranslated",
     FUZZY: "fuzzy",
     TRANSLATED: "translated"}
+
+# Default store priority - used by vfolders atm
+#  - valid range = 0 < n <= 5.0
```

Comment:
Why `5.0`? I agree on setting some upper limit, but at least on MLO we have a vfolder with priority `6.0`, and in the [example in docs](http://docs.translatehouse.org/projects/pootle/en/latest/features/virtual_folders.html#format-for-the-json-file) we have `999.99`.